packages/ui/src/views/repo/repo-branch/components/create-branch-dialog.tsx

```diff
@@ -134,12 +133,15 @@ export function CreateBranchDialog({
                 </Alert.Title>
               </Alert.Container>
             ) : null}
-            <ButtonGroup className="flex justify-end">
-              <Button onClick={onClose} className="text-primary" variant="outline" loading={isCreatingBranch}>
-                {t('views:repos.cancel', 'Cancel')}
-              </Button>
-              <Button type="submit">{t('views:repos.createBranch', 'Create Branch')}</Button>
-            </ButtonGroup>
+
+            <DialogFooter className="-mx-5 -mb-5 mt-5">
```

Comment:
ideally i do not think we should need these styles if we're using `dialogs` the right way.

Reply:
agreed. @srdjan-harness has raised this issue with PP team. we can clean this up once it's fixed.